### PR TITLE
Fix get_file_content() returning bytes

### PR DESCRIPTION
**Pull request by shreyaspapi**
The object [gitlab.v4.objects.ProjectFile](https://python-gitlab.readthedocs.io/en/stable/api/gitlab.v4.html#gitlab.v4.objects.ProjectFile) with the function [decode()](https://python-gitlab.readthedocs.io/en/stable/api/gitlab.v4.html#gitlab.v4.objects.ProjectFile.decode) will return bytes -

`b'---\nspecfile_path: hello.spec\nsynced_files:\n  - hello.spec\n# actions:\n#   post-upstream-clone: "python3 setup.py sdist --dist-dir ."\n# current_version_command: ["python3", "setup.py", "--version"]\n# create_tarball_command: ["python3", "setup.py", "sdist", "--dist-dir", "."]\njobs:\n- job: copr_build\n  trigger: pull_request\n  metadata:\n    targets:\n    - fedora-30-x86_64'`

Expected string - 

```
---
specfile_path: hello.spec
synced_files:
  - hello.spec
# actions:
#   post-upstream-clone: "python3 setup.py sdist --dist-dir ."
# current_version_command: ["python3", "setup.py", "--version"]
# create_tarball_command: ["python3", "setup.py", "sdist", "--dist-dir", "."]
jobs:
- job: copr_build
  trigger: pull_request
  metadata:
    targets:
    - fedora-30-x86_64
```

### CODE REVIEW

File: ogr/services/gitlab/project.py [hunk 1]
```diff
@@ -364,7 +364,7 @@ def get_branches(self) -> List[str]:
     def get_file_content(self, path, ref="master") -> str:
         try:
             file = self.gitlab_repo.files.get(file_path=path, ref=ref)
-            return str(file.decode())
+            return file.decode().decode()
```
Comment (lachmanfrantisek) [inline ogr/services/gitlab/project.py:367]: Really two times?..;)

**Comment by shreyaspapi** [comment]
Added the test for it.

File: ogr/services/gitlab/project.py (hunk 1 above)
Comment (shreyaspapi) [reply ogr/services/gitlab/project.py:367]: Yeah! looks weird the first one is calling [this](https://python-gitlab.readthedocs.io/en/stable/api/gitlab.v4.html#gitlab.v4.objects.ProjectFile.decode).

File: tests/integration/test_gitlab.py [hunk 2]
```diff
@@ -34,6 +34,15 @@ def setUp(self):
 
 
 class GenericCommands(GitlabTests):
+    def test_get_file_content(self):
+        project = self.service.get_project(repo="hello-world", namespace="shreyaspapi")
+
+        file = project.get_file_content(
+            path="test.md", ref="825abfaafb786aa9abbf7093cbe21dbc8d018277"
+        )
```
Comment (lachmanfrantisek) [inline tests/integration/test_gitlab.py:42]: Can we reuse some project we already use (e.g `self.project` -> `packit-service/ogr-tests`)?

**Pull request merged**